src/content/Concerts/06-07Fall.md

```diff
@@ -0,0 +1,41 @@
+---
+# Concert Name (e.g. "Fall Concert")
+concertName: "Fall Concert"
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#F58025"
+# Date YY-MM-DD (for sorting)
+date: "2006-10-00"
+# Season
+season: "2006/07"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: "dvorak"
+# Google Calendar link (if applicable)
+calendar: ""
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Symphony with Love
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = October 2006
+
+Section
+
+Overline = Program
+Work = Ludwig van Beethoven ~ *Coriolan Overture*, op. 62
+Work = Hector Berlioz ~ Roméo Seul from *Roméo et Juliette*
```

Comment:
Hmmmm we need quotes and translations for these

Reply:
maybe just say Scenes from Romeoo....

Reply:
oki

Reply:
what if i have the translations, on the website they had them in english, but wiki had them in french